app/views/publishers/vacancies/job_applications/index.html.slim

```diff
@@ -5,38 +5,21 @@
 .govuk-grid-row
   .govuk-grid-column-full
     - if @vacancy.within_data_access_period?
-      - applications = @job_applications.group_by(&:status)
-      - new_ones = applications.fetch("submitted", []) + applications.fetch("reviewed", [])
-      - rejected = applications.fetch("unsuccessful", [])
-      - shortlisted = applications.fetch("shortlisted", [])
-      - interviewing = applications.fetch("interviewing", [])
```

Comment:
I quite liked this logic being here rather than in the model. It should be simple enough - in fact I'm not sure why we didn't just use group_by here ;-)